PocketKit/Sources/PocketKit/Article/ReadableViewController.swift

```diff
@@ -313,7 +322,15 @@ extension ReadableViewController {
             group.interItemSpacing = .fixed(0)
 
             let section = NSCollectionLayoutSection(group: group)
-            section.contentInsets = Constants.contentSectionContentInsets
+            // Zero out the default leading/trailing contentInsets, but preserve the default top/bottom values.
+            // This ensures each section will be inset horizontally exactly to the readable content width.
+            var contentInsets = section.contentInsets
+            contentInsets.leading = 0
+            contentInsets.trailing = 0
+            contentInsets.top = Constants.contentSectionContentInsets.top
+            contentInsets.bottom = Constants.metaSectionContentInsets.bottom
+            section.contentInsets = contentInsets
+            section.contentInsetsReference = .readableContent
```

Comment:
nit: similar as logic above, should we put this in a private func, so we only have to change in one place

Reply:
im gonna leave that change out for now given it uses different constants,